src/main/java/pl/touk/parking/domain/User.java

```diff
@@ -0,0 +1,50 @@
+package pl.touk.parking.domain;
+
+import org.springframework.data.jpa.domain.AbstractPersistable;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import javax.persistence.Entity;
+
+import static java.util.Collections.singleton;
+
+@Entity
+public class User extends AbstractPersistable<Integer> {
+
+    private String username;
+    private String passwordHash;
+    private String role;
```

Comment:
użytkownik może mieć tylko jedną rolę?

Reply:
Tak założyłem. Jeżeli miałby ich mieć więcej - to mała zmiana w kodzie.